synapse/lib/aha.py

```diff
@@ -537,41 +675,100 @@ def _initCellHttpApis(self):
         self.addHttpApi('/api/v1/aha/provision/service', AhaProvisionServiceV1, {'cell': self})
 
     async def initServiceRuntime(self):
+
         self.addActiveCoro(self._clearInactiveSessions)
 
         if self.isactive:
-            # bootstrap a CA for our aha:network
-            netw = self.conf.get('aha:network')
-            if netw is not None:
 
-                if self.certdir.getCaCertPath(netw) is None:
-                    logger.info(f'Adding CA certificate for {netw}')
-                    await self.genCaCert(netw)
+            # bootstrap a CA for our aha:network
+            netw = self.conf.req('aha:network')
```

Comment:
This will cause an existing AHA cell which isn't setup for doing provisioning or has a dedicated aha:network setup to fail to boot.

Reply:
I think this is acceptable. It should have been a required configuration option to begin with. :+1: